packages/ui-kit/src/components/media-query.ts

```diff
@@ -2,8 +2,13 @@ import styled from '@emotion/styled';
 import { css } from '@emotion/react';
 import { dimensions } from '../design-system';
 
+type MediaQueryProps = {
+  forViewport?: string;
+  hideIfMatch?: boolean;
+};
+
 const MediaQuery = styled.div`
-  ${(props) => {
+  ${(props: MediaQueryProps) => {
```

Comment:
For styled components, you can pass the prop type as the type argument

```ts
styled.div<MediaQueryProps>``
```

https://emotion.sh/docs/typescript#passing-props